lib/topolys_measures/schema/minimal_export.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://json-schema.org/automaticmagic/topolys.json#",
+  "title": "Topolys Measures Edge Properties Schema",
+  "description": "Schema for the Edge Properties for Topolys Measures",
+  
+  "type": "object",
```

Comment:
@rd2 if we want a custom export we can create a new schema

Reply:
@macumber holding specific schemas for distinct tasks (e.g. topolys.json for Topolys arborescence and internal objects; tbd.json for TBD wrapper, and so on) always made more sense to me (that being said, I suspect each schema's "id" would need to be set accordingly)

Reply:
Yes you are right!